internal/log/stdLogger.go

```diff
@@ -0,0 +1,51 @@
+package log
+
+import (
+	"fmt"
+	"io"
+	goLog "log"
+	"os"
+	"runtime"
+)
+
+var stdLogger StdLogger
+
+type StdLogger struct {
+	Logger
+}
+
+func SetOutput(w io.Writer) {
+	goLog.SetOutput(w)
+}
+
+func Output(level string, format string, args ...interface{}) {
+	logMsg := fmt.Sprintf(format, args...)
+	_, file, line, _ := runtime.Caller(3)
+	// timestamp will be provided by goLog
+	goLog.Printf("•%v•%v•%v•%v", level, file, line, logMsg)
+}
+
+func (l *StdLogger) Fatal(format string, args ...interface{}) {
```

Comment:
No need to the pointer here, you are not making use it. Just have it be of the empty struct.
```
func (_ Standard)Fatal(…)
```
This of course applies to the rest of these functions.